components/src/core/ListItemTag/ListItemTag.tsx

```diff
@@ -68,11 +68,8 @@ const listItemTagStyles = (
             color:
                 props.fontColor ||
                 (Color(props.backgroundColor || theme.colors.primary).isLight()
-                    ? // @TODO Currently neutral30 is #414E54 and as per color pallete black[500] is #424E54 Add Figma Variable
-                      // @ts-ignore
-                      BLUIColors.neutral[30]
-                    : // @ts-ignore
-                      BLUIColors.primary[100]),
+                    ? theme.colors.onNeutralShadedContainer
```

Comment:
@huayunh , can you check these color codes are correct? If not please provide me a Figma link of the default color. when we worked on this I didn't have a color code and we just chose close color code if you see removed line 71.